playwright/lib/util.ts

```diff
@@ -201,7 +201,6 @@ const countLiveblogInlineSlots = async (page: Page, isMobile: boolean) => {
 	return await page.locator(locator).count();
 };
 
-// @ts-expect-error -- used when logging uncommented
```

Comment:
We get a warning on every PR with this unused function. Can we leave the `console.warn` in? It's only used for Playwright.